src/Summoner/Question/Data.hs

```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Summoner.Question.Data
+  ( YesNoPrompt(..)
+  , mkYesNoPrompt
+  , mkDefaultYesNoPrompt
+  ) where
+
+import Data.Text (Text)
+
+data YesNoPrompt = YesNoPrompt
+  { yesno_target :: Text
```

Comment:
According to the style guide we are using, these fields should be named `yesNoTarget` and `yesNoPrompt`.